### PR TITLE
libnick, parabolic: update to 2024.12.0

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

Update, add runtime deps

###### Type(s)

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [ ] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: devel/libnick/Portfile [hunk 1]
```diff
@@ -9,16 +9,16 @@ PortGroup           openssl 1.0
 
 boost.version       1.81
 
-github.setup        NickvisionApps libnick 2024.11.1
+github.setup        NickvisionApps libnick 2024.12.0
 revision            0
 categories          devel
 license             GPL-3
-maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+maintainers         {@nlogozzo} openmaintainer
```
Comment (reneeotten) [inline devel/libnick/Portfile:16]: why this change; was that discussed somewhere?

Comment (barracuda156) [reply devel/libnick/Portfile:16]: _This is the upstream._

And yes, see: https://github.com/NickvisionApps/Parabolic/issues/973#issuecomment-2501584866

Comment (reneeotten) [reply devel/libnick/Portfile:16]: okay, but don't expect me to follow upstream discussions to go and figure this out ;)
